pkg/kn/root/root.go

```diff
@@ -92,6 +92,7 @@ Find more information about Knative at: https://knative.dev`, rootName),
 	rootCmd.PersistentFlags().StringVar(&p.KubeAsUID, "as-uid", "", "uid to impersonate for the operation")
 	rootCmd.PersistentFlags().StringArrayVar(&p.KubeAsGroup, "as-group", []string{}, "group to impersonate for the operation, this flag can be repeated to specify multiple groups")
 	flags.AddBothBoolFlags(rootCmd.PersistentFlags(), &p.LogHTTP, "log-http", "", false, "log http traffic")
+	rootCmd.PersistentFlags().BoolVarP(&p.QuietMode, "quiet-mode", "q", false, "run commands in quiet mode")
```

Comment:
I prefer as short names as possible. :)

```suggestion
	rootCmd.PersistentFlags().BoolVarP(&p.QuietMode, "quiet", "q", false, "run commands in quiet mode")
```